shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -248,10 +264,11 @@ private void addJob(URIStatus status, int replication) {
    *
    * @param filePath The {@link AlluxioURI} path to load into Alluxio memory
    * @param replication The replication of file to load into Alluxio memory
+   * @param indexFile file The index file containing all files to load
    */
-  private void distributedLoad(AlluxioURI filePath, int replication)
+  private void distributedLoad(AlluxioURI filePath, int replication, String indexFile)
```

Comment:
Do you check the validity of the index file?

Reply:
RandomAccessFile will throw clear exception in cases the file can't be read.